packages/tomls/src/periphery/configs/tokens/wbtc.toml

```diff
@@ -0,0 +1,120 @@
+#####################################
```

Comment:
- I realised that we also have to decomment code in wbtc.toml under token/
- it would also be nice to add a few tests for wbtc similar to usdc and weth